semeio/jobs/ots/ots.py

```diff
@@ -0,0 +1,597 @@
+from __future__ import print_function
+import sys
+from collections import namedtuple
+import os.path
+from datetime import datetime as dt
+import numpy as np
+from scipy.interpolate import CloughTocher2DInterpolator
+
+from ecl.grid import EclGrid
+from ecl.gravimetry import EclSubsidence
+from ecl.eclfile import Ecl3DKW, EclFile
+
+from ecl.util.geometry import Surface
+
+from .ots_vel_surface import OTSVelSurface
+from .ots_res_surface import OTSResSurface
+from .ots_config import build_schema
+import configsuite
+import yaml
+
+
+def extract_ots_context(configuration):
+    if configuration.eclbase is not None:
+        rstfile_path = "{}.UNRST".format(configuration.eclbase)
+        rstfile = EclFile(rstfile_path)
+        dates = [d.date() for d in rstfile.dates]
+        return dates
+    return []
+
+
+def ots_load_params(input_file):
+    config = None
+    with open(input_file, "r") as fin:
+        config = yaml.safe_load(fin)
+    config = configsuite.ConfigSuite(
+        config,
+        build_schema(),
+        deduce_required=True,
+        extract_validation_context=extract_ots_context,
+    )
+    assert config.valid
+
+    input_data = config.snapshot
+    return input_data, input_data.vintages
+
+
+def write_surface(vintage_pairs, ts, output_dir, type_str):
+    for iv, vp in enumerate(vintage_pairs):
+        d0 = vp[0].strftime("%Y_%m_%d")
+        d1 = vp[1].strftime("%Y_%m_%d")
+        ts[iv].write(output_dir + type_str + "/ots" + "_" + d0 + "_" + d1 + ".irap")
+
+
+def ots_run(parameter_file, verbose=False):
```

Comment:
There is no way to set the `verbose` to True. The caller of this function only passes the configuration file (which, to be honest, I would think could've been parsed up front).

I believe most jobs in semeio uses builtin logger in python. I would think that having a command line argument where one could set `--debug`, and then instead set the logger level would make this much simpler.

All the `if verbose` checks would also go away

Reply:
Well spotted also I agree with having a logger instead. Will fix it.